src/Microsoft.IO.Redist/src/Microsoft.IO.Redist.csproj

```diff
@@ -0,0 +1,237 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AssemblyName>Microsoft.IO.Redist</AssemblyName>
+    <IsPartialFacadeAssembly>false</IsPartialFacadeAssembly>
+    <ProjectGuid>{FEF5811F-ED50-4407-A6B9-885EBD3206FB}</ProjectGuid>
+    <Configurations>netfx-Debug;netfx-Release</Configurations>
+    <DefineConstants>$(DefineConstants);MS_IO_REDIST</DefineConstants>
+    <LangVersion>7.3</LangVersion>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <ClsCompliant>false</ClsCompliant>
+    <UWPCompatible Condition="'$(TargetGroup)' == 'uap' or '$(TargetGroup)' == 'uapaot'">true</UWPCompatible>
+  </PropertyGroup>
+  <ItemGroup Condition="'$(BuildingInsideVisualStudio)' == 'true' or ('$(TargetsWindows)' == 'true' and '$(UWPCompatible)' != 'true')">
+    <Compile Include="..\..\System.IO.FileSystem\src\Microsoft\Win32\SafeHandles\SafeFindHandle.Windows.cs" Link="Microsoft\IO\Win32\SafeHandles\SafeFindHandle.Windows.cs" />
```

Comment:
If you want to try something fun, I bet you could use LinkBase to avoid all this verbosity, see https://github.com/dotnet/sdk/blob/220b9707fe9870975a35a7d032e0a2645302f625/src/Tasks/Microsoft.NET.Build.Extensions.Tasks/Microsoft.NET.Build.Extensions.Tasks.csproj#L58

Reply:
Ooh. I'll play with it.